taskcluster/kinds/dataset/kind.yml

```diff
@@ -119,6 +119,97 @@ tasks:
                     --dataset {dataset}
                     --output_prefix $TASK_WORKDIR/artifacts/{dataset_sanitized}
 
+    bucket:
```

Comment:
suggestion (non-blocking): Add usage of at least one of these new dataset types to https://github.com/mozilla/firefox-translations-training/blob/main/taskcluster/configs/config.ci.yml to ensure that we have some coverage of the importer in CI runs.